man/adoc/bpftrace.adoc

```diff
@@ -3804,6 +3804,15 @@ Default: auto
 
 Colorize the bpftrace log output message. Valid values are auto, always and never.
 
+==== BPFTRACE_SUPPRESS_TEST_SPEW
```

Comment:
Do we really want to expose this to users? I imagine it will only get used in CI test runs.

Reply:
> BTW, if it's only used in CI test runs, we don't need this enhancement at all. Who will check the output if all CI tests pass?

As per your newly added example, I think it reduces a lot of the extraneous log messages when there is a failure and you have to check the CI test runs (less scrolling and noise) - but I'll let @danobi chime in here.